localstack/services/plugins.py

```diff
@@ -117,22 +117,63 @@ class ServiceStateException(ServiceException):
     pass
 
 
+class ServiceLifecycleHook:
+    def on_after_init(self):
```

Comment:
Wondering if over time we may want to pass in the `Service` instance here (or somehow else make it available to the `ServiceLifecycleHook` instance). But we can evolve the interface as we go, based on the requirements that will come up. Good to have this as a starting point... 👍

Reply:
yep sounds good. i think this needs a few iterations :-)